src/python/pants/backend/python/lint/pylint/rules.py

```diff
@@ -1,12 +1,10 @@
 # Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Pylint is unlike the other formatters and linters because we need direct dependencies. This file deserves a closer review than the other linters.